src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -348,9 +348,9 @@ namespace OpenLoco::Vehicles
         void applyBreakdownToTrain();
         void sub_4AF7A4();
         uint32_t getVehicleTotalLength() const;
-        constexpr bool hasFlags(Flags5F flagsToTest) const
+        constexpr bool hasFlags(BreakdownFlags flagsToTest) const
```

Comment:
Could you make this hasBreakdownFlag vehicles have lots of flags so probably best to be quite verbose.